zerver/webhooks/github/view.py

```diff
@@ -132,51 +133,51 @@ def get_member_body(helper: Helper) -> str:
     payload = helper.payload
     return "{} {} [{}]({}) to [{}]({}).".format(
         get_sender_name(payload),
-        payload["action"],
-        payload["member"]["login"],
-        payload["member"]["html_url"],
+        payload["action"].tame(check_string),
+        payload["member"]["login"].tame(check_string),
+        payload["member"]["html_url"].tame(check_string),
         get_repository_name(payload),
-        payload["repository"]["html_url"],
+        payload["repository"]["html_url"].tame(check_string),
     )
 
 
 def get_issue_body(helper: Helper) -> str:
     payload = helper.payload
     include_title = helper.include_title
-    action = payload["action"]
+    action = payload["action"].tame(check_string)
     issue = payload["issue"]
     assignee = issue["assignee"]
     return get_issue_event_message(
         get_sender_name(payload),
         action,
-        issue["html_url"],
-        issue["number"],
-        issue["body"],
-        assignee=assignee["login"] if assignee else None,
-        title=issue["title"] if include_title else None,
+        issue["html_url"].tame(check_string),
+        issue["number"].tame(check_int),
+        issue["body"].tame(check_string),
```

Comment:
An issue `body` is [nullable](https://github.com/octokit/webhooks/blob/v5.6.0/payload-schemas/api.github.com/common/issue.schema.json#L84) (#22111).